modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure.
```

Comment:
per discussion I thought we weren't supporting multiple vcenter datacenters

Reply:
@gnufied ?

Reply:
Yeah we should update the wording. OCP <=4.12 does not support multiple datacenters. We should revisit this doc for 4.13 though, where we do support multiple datacenters properly.

Reply:
Fixed. Thanks! https://github.com/openshift/openshift-docs/pull/52116#issuecomment-1375713882